### DISCUSSION TITLE
Payees

### DISCUSSION THREAD

**Discussion by terminalmage** [NickvisionApps/Denaro#774] [Ideas]
I have QIF exports which contain transactions with both payee and memo fields. However, denaro's ledger format doesn't appear to support payees at all. Every one of my transactions has a payee, but many do not have have a memo field. For example, interest earned/charged doesn't really need a memo field, the transaction category gives you all the info you need to know about that transaction.

Unfortunately this means that after importing, a lot of my transactions simply say "N/A" due to a lack of a memo field.

**Comment by terminalmage** [comment]
I noticed that transactions have a `Notes` area under `Extras`. It seems to me that the `Description` field of the transaction is where the Payee should go, and the `Notes` are where the QIF import's Memo field should go.

Should this be considered a bug in the QIF import, then?

**Reply by nlogozzo** [reply]
I will change the way the QIF import reads the fields to the following algorithm: 

If there are both Payee and Memo fields, Payee will be used for Denaro's `Description` and Memo will be used for `Notes`.
Otherwise, either Payee or Memo will be used for `Description` depending on which of the two are available.

**Comment by nlogozzo** [comment]
See #775
